go/ql/test/query-tests/Security/CWE-643/tst.go

```diff
@@ -10,6 +10,7 @@ package main
 //go:generate depstubber -vendor github.com/jbowtie/gokogiri/xml Node
 //go:generate depstubber -vendor github.com/jbowtie/gokogiri/xpath "" Compile
 //go:generate depstubber -vendor github.com/santhosh-tekuri/xpathparser "" Parse,MustParse
+//go:generate depstubber -vendor github.com/lestrrat-go/libxml2/parser "" ParseString
```

Comment:
This is telling it depstubber to stub a function called `ParseString`, but actually you want it to stub the type `Parser` (it will automatically make stubs for all the methods on that type) and also the function `New`. I think this should work better:
```suggestion
//go:generate depstubber -vendor github.com/lestrrat-go/libxml2/parser Parser New
```

Reply:
(To explain the error message, I think what's happening is that it's failing to find the function `ParseString` in the package `github.com/lestrrat-go/libxml2/parser`, but it is finding it in `github.com/lestrrat-go/libxml2`, so it's stubbing that instead, and then when you try to import `github.com/lestrrat-go/libxml2/parser` you don't have any stubs for it.)

Reply:
👍 only change i had to make was to add a constant that was used by the test!